backend/src/controllers/UserController.js

```diff
@@ -0,0 +1,42 @@
+const connection = require('../database/connection');
+
+module.exports = {
+    async index(request, response) {
+        const user = await connection('user').select('*');
+    
+        return response.json(user);
+    },
+
+    async create(request, response){
+        const { username, password, email, active } = request.body;
+
+        try { 
+            await connection('user').insert({
+                username,
+                password,
+                email,
+                active
+            })
+        } catch (exception) {
+            console.log('Exception: ' + exception);
+            return response.status(400).send();
```

Comment:
Remove over-indentation. These command lines in the "catch" block should be indented at the same level as the "catch" command.

Reply:
Done!